test/lib/cli-test.ts

```diff
@@ -327,13 +333,31 @@ describe('cli', function () {
 
         '.eslint-doc-generatorrc.json': '{ "unknown": true }', // Doesn't match schema.
```

Comment:
Since it's near the new tested you added, can we switch this to this to `JSON.stringify`, it's nicer to deal with an actual JS object than manually construct it in a string.
```suggestion
        '.eslint-doc-generatorrc.json': JSON.stringify({ 'unknown': true }), // Doesn't match schema.
```